pkg/controllers/dynakube/connectioninfo/reconciler.go

```diff
@@ -36,19 +36,32 @@ func NewReconciler(clt client.Client, apiReader client.Reader, scheme *runtime.S
 	}
 }
 
+func (r Reconciler) updateDynakubeStatus(ctx context.Context) {
+	r.dynakube.Status.UpdatedTimestamp = metav1.Now()
+	err := r.client.Status().Update(ctx, r.dynakube)
+	if err != nil {
+		log.Error(err, "could not update dynakube status", "name", r.dynakube.Name)
+	}
+}
+
 func (r *Reconciler) Reconcile(ctx context.Context) error {
+	var activeGateConnectionInfoUpdated bool
 	if !r.dynakube.FeatureDisableActivegateRawImage() {
-		err := r.reconcileActiveGateConnectionInfo(ctx)
+		var err error
+		activeGateConnectionInfoUpdated, err = r.reconcileActiveGateConnectionInfo(ctx)
 		if err != nil {
 			return err
 		}
 	}
 
-	err := r.reconcileOneAgentConnectionInfo(ctx)
+	oneAgentConnectionInfoUpdated, err := r.reconcileOneAgentConnectionInfo(ctx)
 	if err != nil {
 		return err
 	}
 
+	if oneAgentConnectionInfoUpdated || activeGateConnectionInfoUpdated {
+		r.updateDynakubeStatus(ctx)
+	}
```

Comment:
these `bool` values are always `true` if there was no error, so they are not that useful. 
- if there was no change and no error we would still update 

we could either:
- update the status always
- actually check if there was a change in the connectionInfo status
   - [how we do it in the dynakube-controller](https://github.com/Dynatrace/dynatrace-operator/blob/141ae33217b78bf6bdcd054a57cb94777f96313c/pkg/controllers/dynakube/dynakube_controller.go#L158-L158)